client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,121 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import Main from 'components/main';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { getCustomizerUrl } from 'state/sites/selectors';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import DocumentHead from 'components/data/document-head';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class CustomerHome extends Component {
+	static propTypes = {
+		site: PropTypes.object.isRequired,
+		siteId: PropTypes.number.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		customizeUrl: PropTypes.string.isRequired,
+	};
+
+	render() {
+		const { translate, site, customizeUrl } = this.props;
+
+		return (
+			<Main className="customer-home__main is-wide-layout">
+				<PageViewTracker path={ `/customer-home/:site` } title={ translate( 'Customer Home' ) } />
+				<DocumentHead title={ translate( 'Customer Home' ) } />
+				<SidebarNavigation />
+				<div className="customer-home__layout">
+					<div className="customer-home__layout-col">
+						<Card>
+							<CardHeading>{ translate( 'My Site' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Review and update my site' ) }
+							</h6>
+							<div className="customer-home__card-button-pair">
+								<Button href={ site.URL } primary>
+									{ translate( 'View Site' ) }
+								</Button>
+								<Button href={ customizeUrl }>{ translate( 'Edit Homepage' ) }</Button>
+							</div>
+							<ul className="customer-home__card-boxes">
+								<li>{ translate( 'Add a page' ) }</li>
+								<li>{ translate( 'Write blog post' ) }</li>
+								<li>{ translate( 'Customize theme' ) }</li>
+								<li>{ translate( 'Change theme' ) }</li>
+								<li>{ translate( 'Edit menus' ) }</li>
```

Comment:
Hi! I've found a possible matching string that has already been translated **127** times:
`translate( 'Edit Menus' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/customer-home-section)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).